packages/cli-platform-android/src/index.ts

```diff
@@ -11,3 +11,4 @@ export {
 } from './commands/runAndroid';
 export {projectConfig, dependencyConfig} from './config';
 export {getAndroidProject, getPackageName} from './config/getAndroidProject';
+export {startServerInNewWindow} from './commands/buildAndroid/startServerInNewWindow';
```

Comment:
I think we should move `startServerInNewWindow` to the `cli-plugin-metro` package. Because it isn't android specific function. cc. @thymikee